source/synthDriverHandler.py

```diff
@@ -564,3 +576,12 @@ def __init__(self,ID):
 		name=languageHandler.getLanguageDescription(ID)
 		super(LanguageInfo,self).__init__(ID,name)
 
+#: Notifies when a synthesizer reaches an index during speech.
+#: Handlers are called with these keyword arguments:
+#: synth: The L{SynthDriver} which reached the index.
+#: index: The number of the index which has just been reached.
+synthIndexReached = extensionPoints.Action()

```

Comment:
In keeping with the naming scheme for extensionPoints, I think we should change the name here. See the [contrib guide](https://github.com/nvaccess/nvda/wiki/Contributing#identifier-names) or #7606 where the naming convention was initially discussed.

Reply:
I need to look further at how these are used to understand how to rename these to meet the convention. The extension points used in #7606 were essentially informational "X is going to happen" or "X has just happend". My initial interpretation of these extension points is that they don't fit this pattern exactly. One thing that is not clear to me yet, is whether an index is conceptually tied to another command, or if it is independant.